guides/07_streaming/04_conversational-chatbot.md

```diff
@@ -0,0 +1,187 @@
+# Building Conversational Chatbots with Gradio
+
+Tags: AUDIO, STREAMING, CHATBOTS
+
+## Introduction
+
+The next generation of AI user interfaces is moving towards audio-native experiences. Users will be able to speak to chatbots and receive spoken responses in return. Several models have been built under this paradigm, including GPT-4o and [mini omni](https://github.com/gpt-omni/mini-omni).
```

Comment:
Side note but this issue pointed out in the GH Repo for Mini Omni should be fixed, yeah? 

<img width="867" alt="image" src="https://github.com/user-attachments/assets/f9a36a3c-5437-4a88-845e-c557f4a8d327">

Reply:
I think fixed already in the 5.0 branch